src/go/options/common.go

```diff
@@ -49,6 +49,10 @@ type CommonOptions struct {
 	ServiceControlCredentials *IAMCredentialsOptions
 	// Configures the identity used when making requests to backends.
 	BackendAuthCredentials *IAMCredentialsOptions
+
```

Comment:
These should go in config generator options, not common options. None of these flags are needed by the ADS Bootstrapper, they are only needed by the config generator.

Reply:
Similar for the credentials above, but it's a little late to change that.